src/backend/joanie/demo/management/commands/create_dev_demo.py

```diff
@@ -100,6 +98,9 @@ def create_product_credential(self, user, organization, batch_size=1):
                         course=target_course.code, course_run="{course.title}_run2"
                     ),
                 )
+                factories.ProductTargetCourseRelationFactory(
```

Comment:
I would separate this in a separate commit to explain the bug. The other changes are additions/removals but this is a bug fix.